README.md

```diff
@@ -1,17 +1,54 @@
 # Python Community News
+![PCN Logo](.github/assets/PCN%20Logo%20V0.16.jpg)
 
-
-### Watch the stream:
-
+- Check out <https://pythoncommunitynews.com>
 - [Watch the Show on Twitch](https://twitch.tv/kjaymiller)
 - [What the Shown on YouTube](https://youtube.com/c/kjaymiller)
+- https://buttondown.email/python-community-news
+## Topics
 
+The Python
 
-### Subscribe to the Newsletter:
+> Non-Pipable: Any news or topic that do not result in someone needed to `pip install` something.
 
-- https://buttondown.email/python-community-news
+#### Examples of great topics:
+
+- Updates from the [Python Software Foundation (PSF)](https://www.python.org/psf/)
+- Posts from [Pythonistas][^1] that address a concept or thought that could be helpful to other Pythonistas.
```

Comment:
```suggestion
- Posts from Pythonistas[^1] that address a concept or thought that could be helpful to other Pythonistas.
```

Minor change per [the docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#footnotes) to prevent this from rendering as `[Pythonistas][^1]`.